neon_homeassistant_skill/locale/skill.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
This should probably go in `locale/en-us` [per OVOS spec](https://ovos-technical-manual.openvoiceos.org/skill_json/). Then translated metadata can be included with any skill translations